src/Evolution/DgSubcell/Actions/Initialize.hpp

```diff
@@ -119,16 +119,21 @@ struct Initialize {
     const Mesh<Dim> subcell_mesh = fd::mesh(dg_mesh);
```

Comment:
This commit is also a bug. The documentation should be updated instead. This commit prevents subcell from supporting boundary conditions.